env0/validators.go

```diff
@@ -116,7 +116,7 @@ func NewGreaterThanValidator(greaterThan int) schema.SchemaValidateDiagFunc {
 }
 
 func NewOpenTofuVersionValidator() schema.SchemaValidateDiagFunc {
-	return NewRegexValidator(`^(?:[0-9]\.[0-9]{1,2}\.[0-9]{1,2})|1\.6\.0-alpha$`)
+	return NewRegexValidator(`^(?:[0-9]\.[0-9]{1,2}\.[0-9]{1,2})|RESOLVE_FROM_CODE|latest$`)
```

Comment:
should still support alpha and beta versions

Reply:
it's supported. The regex does not end with a $. Therefore, these are matching.

Reply:
then you should drop the `$` in the `latest$`

Reply:
or maybe `(?:^[0-9]\.[0-9]{1,2}\.[0-9]{1,2}(-.+)?$)|^RESOLVE_FROM_CODE$|^latest$`

Reply:
@TomerHeber ^

Reply:
`(?:^[0-9]\.[0-9]{1,2}\.[0-9]{1,2}(?:-.+)?$)|^RESOLVE_FROM_CODE$|^latest$`

Reply:
updated.